package.json

```diff
@@ -18,5 +18,8 @@
   "homepage": "https://github.com/myuw-web-components/myuw-app-bar#readme",
   "devDependencies": {
     "superstatic": "^5.0.1"
+  },
+  "dependencies": {
+    "tota11y": "^0.1.6"
```

Comment:
this may just be a devDependency? not something we want packaged?

Reply:
Oops, yeah I that's what I meant to do. Will update!